contracts/plugins/assets/convex/CvxStableRTokenMetapoolCollateral.sol

```diff
@@ -75,16 +75,36 @@ contract CvxStableRTokenMetapoolCollateral is CvxStableMetapoolCollateral {
         return (low, high, 0);
     }
 
-    /// Should not revert
+    /// Can revert, used by `_anyDepeggedOutsidePool()`
+    /// Should not return FIX_MAX for low
+    /// Should only return FIX_MAX for high if low is 0
+    /// @return lowPaired {UoA/pairedTok} The low price estimate of the paired token
+    /// @return highPaired {UoA/pairedTok} The high price estimate of the paired token
+    function tryPairedPrice()
+        public
+        view
+        virtual
+        override
+        returns (uint192 lowPaired, uint192 highPaired)
+    {
+        // refresh price in oracle if needed
+        (Price memory p, uint48 ts) = rTokenOracle.priceView(IRToken(address(pairedToken)));
+        require(block.timestamp - ts <= rTokenOracle.cacheTimeout(), "call refresh()");
```

Comment:
maybe we move this responsibility into the RTokenPriceOracle and give the function a parameter like `allowStalePrice`? just a thought bc i think we'll end up needing this check in a lot of places and it would be a shame to forget it

Reply:
yup that makes sense

Reply:
Cascades nicely into the RToken Oracle conversation (yes I keep bringing it up, but it reminds me of that haha)